packages/widget-chat/src/components/activity-list/index.js

```diff
@@ -0,0 +1,58 @@
+import React, {Component, PropTypes} from 'react';
+import ActivityItem from '../activity-item';
+
+class ActivityList extends Component {
+  shouldComponentUpdate() {
+    return false;
+  }
+
+  renderActivityItems(activities) {
+    return activities.map((activity) =>
+      <ActivityItem activity={activity} key={activity.id} />
+    );
+  }
+
+  render() {
+    const activities = this.renderActivityItems(this.props.activities);
+    return (
+      <div>
+        <ul>
+          {activities}
+        </ul>
+      </div>
+    );
+  }
+}
+
+ActivityList.propTypes = {
+  activities: PropTypes.array,
+  user: PropTypes.object
+};
+
+ActivityList.defaultProps = {
+  activities: [
```

Comment:
note that this mock doesn't look like the activity stream. I think we did a fair bit in the web client to modify activities to make them make more sense, but ended up footgunning ourselves because the data model diverged from the API in unhelpful ways